core/App/types/oca.ts

```diff
@@ -205,25 +205,38 @@ export class OCABundleResolver implements OCABundleResolverType {
     return this.options.cardOverlayType ?? CardOverlayType.CardLayout11
   }
 
-  public resolveDefaultBundle(credential: CredentialExchangeRecord, language = 'en'): Promise<OCABundle | undefined> {
+  private getDefaultBundle(
+    credDefId?: string,
```

Comment:
IMHO, I consider it good practice use full names for variables and functions, ex `credentialDefinitionId`, `credentialName`, etc. It makes it easier for future developers to understand what's going on.